@ndlib/gatsby-theme-marble/src/utils/helpers.js

```diff
@@ -0,0 +1,60 @@
+import typy from 'typy'
```

Comment:
I think this file (and test) belongs under the Seo directory rather than in a common space since it is currently only answering questions that SEO cares about.

Reply:
I also think we should take the code climate suggestions and you should make a generic get function that gets from the metadata array.  If you have questions let me know.

Reply:
Moved helpers.js to Seo directory, created getFieldValue function and updated unit tests